### PR TITLE
#163917625 A user should be able to create an account

### PR DESCRIPTION
**Pull request by Nduhiu17**
### What does this PR do?
Have the end point to get a specific political party working

### Description of Task to be completed?
User should get a specific political party

### How should this be manually tested?
* Clone the repo by git clone by : git clone https://github.com/Nduhiu17/politico-server.git
* Navigate into the project by: cd political-server
* Install a virtual environment by: python3.6 -m venv virtual
* Install dependancies by: pip install -r requirements.txt
* Set server's secret key by:
export SECRET_KEY='put-your-secret-key-here'
* Export your database url by:
export DATABASE_URL="postgres://yourusername:yourpassword@localhost:5432/your-test-db-name"
* Run the server by: python manage.py server
* Hit the following endpoint with postman: http://127.0.0.1:5000/api/v2/auth/signup['POST']

### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/163917625

### Screenshot
screenshot from 2019-02-13 20-32-43

### CODE REVIEW

File: app/V2/auth/models.py [hunk 1]
```diff
@@ -1 +1,80 @@
...
+
+cursor = Database.connect_to_db()
+Database.create_users_tables()
+
+
+class User:
+    """Class that models a user"""
+
+    def __init__(self, id, firstname, lastname, othername, email, phonenumber, passporturl, password,
+                 date_created,
```
Comment (Quantum-35) [inline app/V2/auth/models.py]: In line 16 use *args instead of more than 4 parameters.

File: app/V2/auth/models.py [hunk 2]
```diff
@@ -1 +1,80 @@
...
+        self.lastname = lastname
+        self.othername = othername
+        self.email = email
+        self.phonenumber = phonenumber
+        self.passporturl = passporturl
+        self.password = password
+        self.date_created = date_created
+        self.date_modified = date_modified
+
+    def save(self, firstname, lastname, othername, email, phonenumber, passporturl, password):
```
Comment (Quantum-35) [inline app/V2/auth/models.py]: same here

File: app/V2/auth/views.py [hunk 3]
```diff
@@ -1 +1,98 @@
...
+    if not request.json or not 'passporturl' in request.json:
+        return make_response(jsonify({
+            "status": 400,
+            "error": "Passporturl is required"
+        }), 400)
+
+    if not request.json or not 'password' in request.json:
+        return make_response(jsonify({
+            "status": 400,
+            "error": "Password is required"
```
Comment (Quantum-35) [inline app/V2/auth/views.py]: you have a lot of if statements. Could you find a way to reduce them?

File: app/V2/auth/models.py (hunk 1 above)
Comment (Nduhiu17) [reply app/V2/auth/models.py]: Noted.I will work on it.

File: app/V2/auth/models.py (hunk 2 above)
Comment (Nduhiu17) [reply app/V2/auth/models.py]: Cool

File: app/V2/auth/views.py (hunk 3 above)
Comment (Nduhiu17) [reply app/V2/auth/views.py]: Yes I will do.

**Review by Quantum-35 — APPROVED** [review]
Cool. Looks good to merge.

**Pull request merged**